.gitignore

```diff
@@ -22,3 +22,5 @@ retworkx/*pyd
 *.jpg
 **/*.so
 retworkx-core/Cargo.lock
+.DS_Store
+**/.DS_Store
```

Comment:
A bit surprising that `.DS_Store` is not in there, but it should be fine. I think we only need the second instance though, right? It makes the first one redundant